src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,69 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
 public class WorkWithFile {
 
+    private String createReport(int supply, int buy, int result) {
+        return "supply," + supply + System.lineSeparator() + "buy," + buy
+                + System.lineSeparator() + "result," + result;
+    }
+
+    private String readFromFile(String fileName) {
+        try {
+            return Files.readString(Path.of(fileName));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't correctly read data from file " + fileName, e);
+        }
+    }
+
+    private void writeToFile(String report, String fileName) {
+        BufferedWriter bufferedWriter = null;
+        try {
+            bufferedWriter = new BufferedWriter(new FileWriter(fileName));
+            bufferedWriter.write(report);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write to the file", e);
+        } finally {
+            if (bufferedWriter != null) {
+                try {
+                    bufferedWriter.close();
+                } catch (IOException e) {
+                    throw new RuntimeException("Can't close the file", e);
+                }
+            }
+        }
+    }
+
+    private int getNumberFromString(String string) {
+        StringBuilder tempNumberString = new StringBuilder("");
+        char[] charArray = string.split(",")[1].toCharArray();
+        for (int i = 0; i < charArray.length - 1; i++) {
+            tempNumberString.append(charArray[i]);
+        }
+        int tempNumber = Integer.parseInt(tempNumberString.toString());
```

Comment:
the Travis fails because when tested on an empty string, your code throws NumberFormatException on this line

see more here: https://travis-ci.com/github/mate-academy/jv-work-with-file/builds/207173828
the reports are available on the "Checks" page in your PR